Mathlib/Analysis/Calculus/FDeriv/Symmetric.lean

```diff
@@ -441,37 +465,73 @@ theorem second_derivative_symmetric_of_eventually {f' : E → E →L[𝕜] F} {x
 
 /-- If a function is differentiable, and has two derivatives at `x`, then the second
 derivative is symmetric. -/
-theorem second_derivative_symmetric {f' : E → E →L[𝕜] F} {f'' : E →L[𝕜] E →L[𝕜] F} {x : E}
+theorem second_derivative_symmetric [IsRCLikeNormedField 𝕜]
+    {f' : E → E →L[𝕜] F} {f'' : E →L[𝕜] E →L[𝕜] F} {x : E}
     (hf : ∀ y, HasFDerivAt f (f' y) y) (hx : HasFDerivAt f' f'' x) (v w : E) : f'' v w = f'' w v :=
   second_derivative_symmetric_of_eventually (Filter.Eventually.of_forall hf) hx v w
 
-/-- If a function is `C^2` at a point, then its second derivative there is symmetric. -/
-theorem ContDiffAt.isSymmSndFDerivAt {n : WithTop ℕ∞} (hf : ContDiffAt 𝕜 n f x) (hn : 2 ≤ n) :
-    IsSymmSndFDerivAt 𝕜 f x := by
-  intro v w
-  apply second_derivative_symmetric_of_eventually (f := f) (f' := fderiv 𝕜 f) (x := x)
-  · obtain ⟨u, hu, h'u⟩ : ∃ u ∈ 𝓝 x, ContDiffOn 𝕜 2 f u :=
-      (hf.of_le hn).contDiffOn (m := 2) le_rfl (by simp)
-    rcases mem_nhds_iff.1 hu with ⟨v, vu, v_open, xv⟩
-    filter_upwards [v_open.mem_nhds xv] with y hy
-    have : DifferentiableAt 𝕜 f y := by
-      have := (h'u.mono vu y hy).contDiffAt (v_open.mem_nhds hy)
-      exact this.differentiableAt one_le_two
-    exact DifferentiableAt.hasFDerivAt this
-  · have : DifferentiableAt 𝕜 (fderiv 𝕜 f) x := by
-      apply ContDiffAt.differentiableAt _ le_rfl
-      exact hf.fderiv_right hn
-    exact DifferentiableAt.hasFDerivAt this
+open scoped Classical in
+variable (𝕜) in
+/-- `minSmoothness 𝕜 n` is the minimal smoothness exponent larger than `n` for which one can do
```

Comment:
"larger than" tends to evoke associations "larger at not equal" (as opposed to e.g. "at least").
This may be entirely subjective, but I would personally prefer "at least n".